CustomKnight/Skin/Base/Skinable.cs

```diff
@@ -166,6 +166,10 @@ public abstract class Skinable_Sprite : Skinable
                 Reset();
             }
         }
+        public Sprite CreateSpritewithppu(Texture2D tex,float ppu)
+        {
+            return Sprite.Create(tex, new Rect(0, 0, tex.width, tex.height), new Vector2(0.5f, 0.5f), ppu);//We need ppu
+        }
```

Comment:
seems like a utility function better fit for `Satchel.SpriteUtils` possibly making this method better <https://github.com/PrashantMohta/Satchel/blob/a72167380138fdd75133534380bceb4a6c28322d/Utils/SpriteUtils.cs#L8>